content/docs/csidriver/features/powerscale.md

```diff
@@ -198,6 +198,10 @@ spec:
 
 > Starting from CSI PowerScale driver version 2.2, different isi paths can be used to create PersistentVolumeClaim from VolumeSnapshot.This means the isi paths of the new volume and the VolumeSnapshot can be different.
 
+>**Note:** When the access mode of ReadWriteMany is used, a writable snapshot is created using the read only snapshot as the source. Creation of snapshots from PVCs which are created from snapshots is not supported.
+
+>**Note:** Older releases of OneFS limit the number of writable snapshot to a maximum of thirty per cluster. Newer releases of OneFS increased the limit to 1024 writable snapshots per cluster. Consult your Dell PowerStore Technical representative if you need to increase the number of writable snapshots.

```

Comment:
Change "snapshot" to "snapshots" line 203

Reply:
Fixed.